src/main/java/lesson9Text/TextAnalyzer.java

```diff
@@ -0,0 +1,23 @@
+package lesson9Text;
+
+public class TextAnalyzer {
+
+    public static String analyzeText(String text){
+        if (text.equals("")){
```

Comment:
A co jeśli text będzie nullem? ;>

Reply:
gdy próbuje ustalić taki warunek, intelliJ mówi, że text nigdy nie będzie nullem (text == null is always false)

Reply:
poprawione

Reply:
IntelliJ widzi, że w żadnej sytuacji nie podajesz do tej metody wartości null, w rzeczywistości jednak warto się przed taką sytuacją zabezpieczyć.